gtsam/hybrid/tests/testHybridBayesNet.cpp

```diff
@@ -552,19 +553,29 @@ TEST(HybridBayesNet, Sampling) {
   EXPECT_LONGS_EQUAL(2, average_continuous.size());
   EXPECT_LONGS_EQUAL(num_samples, discrete_samples.size());
 
-  // Regressions don't work across platforms :-(
-  // // regression for specific RNG seed
-  // double discrete_sum =
-  //     std::accumulate(discrete_samples.begin(), discrete_samples.end(),
-  //                     decltype(discrete_samples)::value_type(0));
-  // EXPECT_DOUBLES_EQUAL(0.477, discrete_sum / num_samples, 1e-9);
-
-  // VectorValues expected;
-  // expected.insert({X(0), Vector1(-0.0131207162712)});
-  // expected.insert({X(1), Vector1(-0.499026377568)});
-  // // regression for specific RNG seed
-  // EXPECT(assert_equal(expected, average_continuous.scale(1.0 /
-  // num_samples)));
+  // regression for specific RNG seed
+  double discrete_sum =
+      std::accumulate(discrete_samples.begin(), discrete_samples.end(),
+                      decltype(discrete_samples)::value_type(0));
+#if __APPLE__
```

Comment:
I don’t see differences between these different paths??

Reply:
I am using the CI to understand the differences in the results since I can avoid instrumenting individual platform runs manually. I'll request a review when this is ready.